Flag --strict-markers is not available in our version of pytest. This can be
removed once pytest is updated to version 4.5.0 or newer.

Because pytest-benchmark is not available in Solaris, tests using it are skipped.

--- lazy-object-proxy-1.6.0/setup.cfg
+++ lazy-object-proxy-1.6.0/setup.cfg
@@ -23,7 +23,6 @@ markers =
 	xfail_simple: Expected test to fail on the `simple` implementation.
 addopts = 
 	-ra
-	--strict-markers
 	--ignore=docs/conf.py
 	--ignore=setup.py
 	--ignore=ci
--- lazy-object-proxy-1.6.0/tests/test_lazy_object_proxy.py
+++ lazy-object-proxy-1.6.0/tests/test_lazy_object_proxy.py
@@ -1767,6 +1767,7 @@ def test_garbage_collection_count(lop):
     assert count == sys.getrefcount(obj)
 
 
+@pytest.mark.skip("pytest-benchmark is not available on Solaris")
 @pytest.mark.parametrize("name", ["slots", "cext", "simple", "django", "objproxies"])
 def test_perf(benchmark, name, lop_loader):
     implementation = lop_loader(name)
@@ -1836,6 +1837,7 @@ def prototype(request):
         return LocalsCachedPropertyProxy
 
 
+@pytest.mark.skip("pytest-benchmark is not available on Solaris")
 @pytest.mark.benchmark(group="prototypes")
 def test_proto(benchmark, prototype):
     obj = "foobar"
